contracts/artifacts/IUniswapLikePair.json

```diff
@@ -0,0 +1,1431 @@
+{
```

Comment:
Where does this file come from? Our other contracts abis are defined in `contracts/src/bin/vendor.rs` but this one is not.

Reply:
Oh... So I am supposed to provide a source for it here? I had just pasted it in there ad I also did with `IUniswapLikeRouter` am I supposed to do something different?

Reply:
This is just an interface contract that I got from here:

@uniswap/v2-core/contracts/interfaces/IUniswapV2Pair.sol

Reply:
If it's not in a npm package or something like that it's fine I guess. I was just curious. Not super important, but I think it would be nice to minimize the json though since we won't be editing it by hand. The vendored artifacts are all only one line without whitespaces.

Reply:
I guess I'll look and see if I can do that. I suppose removing all the white space would clear that up.

Reply:
Good catch @vkgnosis . I believe this artefact can be vendored from npm just like https://github.com/gnosis/oba-services/blob/10e301a3122b5437cc4e29065d4b74c3085441c8/contracts/src/bin/vendor.rs#L30-L33. It looks like the second value in the `NPM_CONTRACTS` lists can be used to create filenames with arbitrary name overrides (e.g. IUniswapLikePair.json)

There is also a README with more instructions.

Reply:
https://github.com/gnosis/gp-v2-services/pull/559 for reference (there were some issues with the vendor script which require to update the patched ethcontract version)